### ISSUE TITLE
FasterRCNN and ROIAlign are Non-Functional in Master

### ISSUE THREAD

**Issue by VarunS1997** [keras-team/keras-cv#2449]
### Current Behavior:
While investigating #2388, I learned that the current tests for FasterRCNN all fail. Further, the functionality of ROIAlign doesn't appear to be working as is. Further, ROIAlign _may_ be ready for deletion as per TODO.

### Expected Behavior:
Tests for FasterRCNN as well as a basic usage of ROIAlign should both work.

### Steps To Reproduce:
Run FasterRCNN tests

### Version:
HEAD

### Anything else:
N/A

**Comment by sineeli** [comment]
@VarunS1997 

Seems to be `ops.take`  missing `axis` argument in ROIAlign code so tried it and it seems to be working as expected. 

Compared same with Tensorflow Model Garden as the pipelines of MaskRCNN and FasterRCNN are stable in TFM.

Tested [gist](https://colab.sandbox.google.com/gist/sineeli/ee626544ad4765ec4a399ad7c1f1df88/faster-r-cnn-keras3-port.ipynb)

**Comment by sachinprasadhs** [comment]
Thanks for reporting the issue! We have consolidated the development of KerasCV into the new **KerasHub** package, which supports image, text, and multi-modal models. Please read [the announcement](https://github.com/keras-team/keras-hub/issues/1831). `KerasHub` will support all the core functionality of `KerasCV`.

`KerasHub` can be installed with `!pip install -U keras-hub`. Documentation and guides are available at [keras.io/keras_hub](https://keras.io/keras_hub/).

With our focus shifted to KerasHub, we are not planning any further development or releases in KerasCV. If you encounter a KerasCV feature that is missing from KerasHub, or would like to propose an addition to the library, please file [an issue with KerasHub](https://github.com/keras-team/keras-hub/issues).
